plugin/src/adapter/kotlin/org/jlleitschuh/gradle/ktlint/worker/LintErrorResult.kt

```diff
@@ -10,9 +9,9 @@ import java.io.Serializable
  * @param lintedFile file that was checked by KtLint
  * @param lintErrors list of found errors and flag indicating if this error was corrected
  */
-internal data class LintErrorResult(
```

Comment:
I presume this is because kotlin doesn't see this class as being a part of the same project? There is a way to configure the kotlin compiler "friend path" but I don't know if that API is exposed via Gradle. So this is reasonable.

But if you're curious, feel free to look into it.

Reply:
Oh hey... Look what I found. A ticket I opened 5 years ago on this same topic: https://youtrack.jetbrains.com/issue/KT-20760

Reply:
I was able to get friend paths working, but making this internal will also require KtLintInvocation to be internal, and then the work action can't invoke it without also being internal, and I assume those can't be internal

Reply:
No worries